python/tvm/contrib/hexagon/pytest_plugin.py

```diff
@@ -56,14 +56,19 @@ def _compose(args, decs):
 requires_hexagon_toolchain = tvm.testing.requires_hexagon(support_required="compile-only")
 
 
-@pytest.fixture(scope="session")
 def android_serial_number() -> Optional[str]:
```

Comment:
This function no longer returns `Optional[str]`.  Please make it return `list[str]` for all cases, for consistency (including `[]` if there is no device of any kind).